cst438_gradebook/src/main/java/com/cst438/domain/Course.java

```diff
@@ -1,5 +1,8 @@
 package com.cst438.domain;
 
+import org.hibernate.annotations.LazyCollection;
+import org.hibernate.annotations.LazyCollectionOption;
+
```

Comment:
LazyCollection is used correctly here due to the start of the initialized in the private string and added as a fetched type as well as it's own string stating an output of False.